### PR TITLE
LWIMIQ by Harpreet, 04-24-2017

### PR DESCRIPTION
**Pull request by h-kataria**
@syncrou @cybette @imtayadeway @JPrause please review

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @h-kataria

File: site/_posts/2017-04-24-Squashing-Bugz.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Cleanup
+
+Aside from some new additions & lots of bug fixing, we also deleted some unwanted code from our repos.
+
+[MartinP](https://github.com/martinpovolny) Remove duplicate flash_msg_div
+([#1015](https://github.com/ManageIQ/manageiq-ui-classic/pull/1015)) & 
+([#1016](https://github.com/ManageIQ/manageiq-ui-classic/pull/1016)) 
+
+[Brandon](https://github.com/bdunne) Removal of unsused Global Variables from manageiq-ui-repo ([#969](https://github.com/ManageIQ/manageiq-ui-classic/pull/969))
```
Comment (imtayadeway) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: Not sure, but is this supposed to be manageiq-ui-classic? or manageiq (as opposed to manageiq-ui-repo)?

Comment (h-kataria) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: @imtayadeway this link is correct, it is supposed to be a link to a PR in manageiq-ui-classic repo: https://github.com/ManageIQ/manageiq-ui-classic/pull/969

Comment (imtayadeway) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: @h-kataria yes, but you've called it `manageiq-ui-repo` - is that the convention for referring to that?

File: site/_posts/2017-04-24-Squashing-Bugz.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+[Tzu-Mainn](https://github.com/tzumainn) Remove code that creates phantom object when creating Cloud Volume & Tenant ([#14766](https://github.com/ManageIQ/manageiq/pull/14766))
+
+[Jozef](https://github.com/jzigmund) Removing unused host trait ([#14598](https://github.com/ManageIQ/manageiq/pull/14598))
+
+[Nimrod](https://github.com/nimrodshn) Removed duplicate report ([#14515](https://github.com/ManageIQ/manageiq/pull/14515))
+
+## Wrapping up
+
+That's all i have to share with you folks. Stay tuned for next week's LWIMIQ post, by
+[Drew](https://github.com/syncrou).
```
Comment (JPrause) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: Actually,..next week's post will be by @d-m-u

Comment (h-kataria) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: oops sorry, will fix that

File: site/_posts/2017-04-24-Squashing-Bugz.md (hunk 1 above)
Comment (h-kataria) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: @imtayadeway fixed, good catch!

**Comment by JPrause** [comment]
LGTM! Nice job @h-kataria

**Comment by syncrou** [comment]
@h-kataria - This looks 👍 to me!  - I don't think I've ever read a blog post with as many references to PR's in each section.  Great overview of the week across all of our repos!

Thanks for a great read!

**Comment by JPrause** [comment]
@cybette seems we've got all reviews/edits completed. When you have a chance, can you do the final review.

File: site/_posts/2017-04-24-Squashing-Bugz.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+title: "Last Week in ManageIQ: Squashing Bugz"
+author: h-kataria
+date: 2017-04-24
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello there! This is [Harpreet Kataria](https://github.com/h-kataria), my first contribution at LWIMIQ blog post, I want to share some updates on what has been happening in ManageIQ world this past week. 
+Team ManageIQ has been busy squashing bugz rapidly to make Fine release finer and stable. After all bugz are bugz and must be squashed.
```
Comment (cybette) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: "finer and more stable" (make them both comparatives)

File: site/_posts/2017-04-24-Squashing-Bugz.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+Several fixes were made in Embedded Ansible area and Ansible Services area, see list of PRs in manageiq-ui-classic & manageiq repos for full details on these.
+
+[MartinH](https://github.com/himdel) made performance improvements with Notifications ([#611](https://github.com/ManageIQ/manageiq-ui-classic/pull/611))
+
+[Brian](https://github.com/bmclaughlin) Fix Group/Role name links on User Details page ([#655](https://github.com/ManageIQ/manageiq-ui-classic/pull/655))
+
+[Aparna](https://github.com/AparnaKarve) Quadicon links fixed on Tagging edit screen ([#1041](https://github.com/ManageIQ/manageiq-ui-classic/pull/1041))
+
+[Zahi](https://github.com/zakiva) Add Container entities to TAG_CLASSES ([#14535](https://github.com/ManageIQ/manageiq/pull/14535/files))
```
Comment (cybette) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: Do you mean to link to the /files section of the PR? (just checking)

File: site/_posts/2017-04-24-Squashing-Bugz.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+Aside from some new additions & lots of bug fixing, we also deleted some unwanted code from our repos.
+
+[MartinP](https://github.com/martinpovolny) Remove duplicate flash_msg_div
+([#1015](https://github.com/ManageIQ/manageiq-ui-classic/pull/1015)) & 
+([#1016](https://github.com/ManageIQ/manageiq-ui-classic/pull/1016)) 
+
+[Brandon](https://github.com/bdunne) Removal of unsused Global Variables from manageiq-ui-classic repo ([#969](https://github.com/ManageIQ/manageiq-ui-classic/pull/969))
+
+[Roman](https://github.com/rblanco) Removed unused & invalid provider documentation link ([#1002](https://github.com/ManageIQ/manageiq-ui-classic/pull/1002))
```
Comment (cybette) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: I think Roman's github account is https://github.com/romanblanco , not rblanco

File: site/_posts/2017-04-24-Squashing-Bugz.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+[Tzu-Mainn](https://github.com/tzumainn) Remove code that creates phantom object when creating Cloud Volume & Tenant ([#14766](https://github.com/ManageIQ/manageiq/pull/14766))
+
+[Jozef](https://github.com/jzigmund) Removing unused host trait ([#14598](https://github.com/ManageIQ/manageiq/pull/14598))
+
+[Nimrod](https://github.com/nimrodshn) Removed duplicate report ([#14515](https://github.com/ManageIQ/manageiq/pull/14515))
+
+## Wrapping up
+
+That's all i have to share with you folks. Stay tuned for next week's LWIMIQ post, by
```
Comment (cybette) [inline site/_posts/2017-04-24-Squashing-Bugz.md]: Just nitpicking, 'I' should be in upper case :sweat_smile:

**Comment by h-kataria** [comment]
@cybette please re-review, addressed your feedback

**Pull request merged**